paritytech/polkadot-api
# feat: add `@polkadot-api/light-client-extension-helpers`

# How to test?

Launch 4 terminals, then
```
pnpm build
cd packages/light-client-extension-helpers && pnpm dev
cd examples/extension-dapp && pnpm dev
cd examples/extension && pnpm dev
cd examples/extension && pnpm start
```

- Browse `http://localhost:5173/`, open the console and click the buttons to see `console.log` messages
- Browse `chrome://extensions/`, enable `Developer mode`, inspect `service worker`
- Click the extension icon, in the context menu click `Options` and in the options tab open the console and see `console.log` messages
